### DISCUSSION TITLE
Basic setup question

### DISCUSSION THREAD

**Discussion by NODRT** [Vinzent03/obsidian-git#511] [Problems] [answered]
I cannot figure out the basic values to enter. I have GIT (gitea) installed and working. I have pushed my notes to the repo. All of that is in place.

On the Obsidian side, I don't know if my values are correct. The error just says:

> git obsidian error: Cannot run git command

This value maps to my git.exe file:
![image](https://github.com/denolehov/obsidian-git/assets/7833989/23a02aaa-d039-4332-a486-20abe8c3568e)
![image](https://github.com/denolehov/obsidian-git/assets/7833989/175523bf-afec-4fea-beb2-07b3951f2fde)

The values here map to the folder below
![image](https://github.com/denolehov/obsidian-git/assets/7833989/941e151d-1bca-4a3a-9098-4295243ecc80)
![image](https://github.com/denolehov/obsidian-git/assets/7833989/46e1a615-f366-4567-89e1-70b08a516b0b)

Are these wrong? Where should I be pulling these values from?

**Comment by Vinzent03** [comment, marked as answer]
Please remove your "Custom base path" and "Custom Git directory path" setting and append `git.exe` to your "Custom git binary path" setting. So that it points to a file instead of a directory.

**Reply by NODRT** [reply]
Looks like that worked! I thought I tried everything, but I guess I didn't try that. Thank you!
